tests/data_processing/test_get_flight_stats.py

```diff
@@ -0,0 +1,151 @@
+import json
+import unittest
+
+import pandas as pd
+import pandas.testing as pd_testing
+
+from src.data.get_flight_stats import FlightStats
+from src.utils.json_converter import DataConverter
+
+
+class TestFlightStats(unittest.TestCase):
+    FLIGHT_COUNT = [
+        'total_arrivals',
+        'total_departures'
+    ]
+
+    def assertDataframeEqual(self, a: pd.DataFrame, b: pd.DataFrame, msg: None = None) -> None:
+        try:
+            pd_testing.assert_frame_equal(a, b)
+        except AssertionError as e:
+            raise self.failureException(msg) from e
+
+    def setUp(self) -> None:
+        self.addTypeEqualityFunc(pd.DataFrame, self.assertDataframeEqual)
```

Comment:
Should this be `setUpClass` instead as I presume this only needs to be done once?

Reply:
It should but I cant figure out how to get it to run